### PR TITLE
Add message to inform players how to exit the coach slot

### PR DESCRIPTION
**Pull request by nickdnk**
It seems some players are confused about how to "uncoach" once assigned as a coach.

This simply adds a message directed at the coaching player only, that they can `!coach` to go back to being a player. Only sent if still in warmup, as that's when it applies.

@ Translators: The following needs to be translated. Just comment with the language you know and the translated value:

`Type !coach to exit the coach slot.`

### CODE REVIEW

File: documentation/docs/translations.md [hunk 1]
```diff
@@ -161,6 +161,7 @@ end with a full stop as this is added automatically.
 | `ReadyToResumeVetoInfoMessage`              | Type _!ready_ when you are ready to resume the veto.                                                                                                                     | Chat       |
 | `MatchConfigLoadedInfoMessage`              | Loaded match config.                                                                                                                                                     | Chat       |
 | `MoveToCoachInfoMessage`                    | You were moved to the coach position as your team is full.                                                                                                               | Chat       |
+| `CoachingExitInfo`                          | Type _!coach_ to exit the coach slot.                                                                                                                                    | Chat       |
```
Comment (SidiBecker) [inline documentation/docs/translations.md:164]:
```
"CoachingExitInfo"
    {
        "pt"            "Digite {1} para sair da vaga de coach."
    }
```

**Comment by Iwhite67** [comment]
Type !coach to exit the coach slot.
Taper !coach pour sortir du mode mode coach

**Comment by Iwhite67** [comment]
> > Type !coach to exit the coach slot. Taper !coach pour sortir du mode mode coach
> 
> Are you sure you want 2x `mode` in that sentence?

Only one

**Comment by Apfelwurm** [comment]
Type !coach to exit the coach slot.
Gebe !coach ein um die Rolle des Trainers abzugeben.

While writing that sentence, i figured, i have to make changes in the rest of the german translation as well, because even if the term coach is known in germany, the right way to translate it is "Trainer".

**Comment by nickdnk** [comment]
@Saph1s Can you add the Russian?

**Review by Saph1s — COMMENTED** [review]
Russian:

`"ru"            "Введите {1}, чтобы покинуть слот тренера."`

**Comment by nickdnk** [comment]
Confirmed working as intended; only translations needed now.

**Pull request merged**